### ISSUE TITLE
Document how large a site WCAG-EM was designed to document

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-wcag-em-report-tool#106]
Somewhere in the project guide there should be project limits listed somewhere.  

I'm sure that WCAG-EM would fail (even from a UI perspective) if you tried to document 1000 pages, but could it do 200? 

At what point will browser memory be problematic. 

If you're importing a 1000 page EARL file, will the browser be able to load it? I don't know how much data is regularly part of an EARL report, but I can see it quickly adding up to a large page. 

It would be nice to know that WCAG-EM has been tested to work with 250 pages, but not more (or whatever).
